connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResource.java

```diff
@@ -75,16 +75,17 @@ public ConnectorsResource(Herder herder) {
 
     @GET
     @Path("/")
-    public Collection<String> listConnectors() throws Throwable {
+    public Collection<String> listConnectors(final @QueryParam("forward") Boolean forward) throws Throwable {
```

Comment:
Do you want `@DefaultValue` or whatever the appropriate annotation is? Same for rest of the `forward` flags.